
// Generated by hack/generate-e2e-test.py
package test.e2e.core.function;

import static org.junit.Assert.fail;
import org.junit.Test;
import test.e2e.core.E2eTestBase;

public class QuadraticVertex1Test extends E2eTestBase {

  public QuadraticVertex1Test() {
    coreCodeTemplate = "# quadraticVertex 1 https://processing.org/reference/quadraticVertex_.html\n\nnoFill()\nstrokeWeight(4)\nbeginShape()\nvertex(20, 20)\nquadraticVertex(80, 20, 50, 50)\nendShape()\n\nsaveFrame(\"%s\")\nexit()\n";
    referenceURI = "https://processing.org/reference/images/quadraticVertex_0.png";
  }

  @Test
  public void test() {
    try {
      defaultOperation();
    } catch (Exception exception) {
      System.err.println(exception);
      fail("Should not have thrown any exception");
    }
  }
}
